app/helpers/application_helper.rb

```diff
@@ -1,2 +1,12 @@
 module ApplicationHelper
+	def mobile_device
```

Comment:
This is a good way to implement a mobile check.

Reply:
Agreed, this is a good idea. Might be hard to apply to the overall formatting though.